uwb_simulation:
  noise:
    mu: 0 #0.5
    sigma: 0 #0.5
  nAnchors: 5
  anchors: [
    [-1, 2, 2],
    [2, -6, 2],
    [-2, -12, 2],
    [-12, -12, 2],
    [-12, -2, 2]
  ]
